UHResidentInformationAPI/V1/Infrastructure/Address.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("property")]
+    public class Address
+    {
+       [Column("prop_ref")]
+        [MaxLength(9)]
```

Comment:
Looks like there might be some indentation to adjust here? Probably caught by the linter, too. 🙂